src/ComputerPlayer.java

```diff
@@ -3,7 +3,8 @@ public class ComputerPlayer extends Player{
 	private final int MIN_POINTS = 15;
 	
 	public ComputerPlayer(){
-		super("R2D2");
+		super("CPU");
+		myName = "R2D2";
```

Comment:
This only fixes it for ComputerPlayer.  Setting the name isn't the ComputerPlayer's job - it is the Player's job.